src/App.js

```diff
@@ -171,9 +173,15 @@ class App extends React.Component {
       return
     }
 
-    this.setState({ appsStatus: APPS_STATUS_LOADING, apps: [] })
+    // Reset the DAO state
+    this.setState({
+      appsStatus: APPS_STATUS_LOADING,
+      apps: [],
+      daoAddress: '',
```

Comment:
Would it make sense to namespace these properties (and potentially new ones when/if they arrive) into a `DAO`/`currentDao` property?

```js
{
  ...
  DAO: {
    address: '',
    domain: '',
  }
}
```

Reply:
Yes good idea!